ANMN/LTSP/TSaggregator/geoserverCatalog_generic.py

```diff
@@ -0,0 +1,112 @@
+## read all ANMN/ABOS files names form the geoserver
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+geoserverCatalog.py
+
+Collect files names from the AODN geoserver according to several conditions
+Output a list of urls and optionally write into a text file
+
+"""
+
+
+import sys
+import argparse
+from datetime import datetime, timedelta
+from distutils.util import strtobool
```

Comment:
`timedelta` and `strtobool` are not used in the script, so don't need to be imported

Reply:
remnant of previous version. Removed.